services/wallet/gemini_test.go

```diff
@@ -0,0 +1,207 @@
+package wallet
+
+import (
+	"testing"
+
+	"github.com/brave-intl/bat-go/libs/clients/gemini"
+	should "github.com/stretchr/testify/assert"
+)
+
+func Test_getIssuingCountry(t *testing.T) {
```

Comment:
Nit: The name of this test should be `TestGetIssuingCountry` or `TestGeminix_GetIssuingCountry`, as underscores in test names normally separate a type from a method that is being tested, if a test is for a specific method.

```suggestion
func TestGeminix_GetIssuingCountry(t *testing.T) {
```